lib/resources/docker_container.rb

```diff
@@ -78,7 +78,7 @@ def repo
     end
 
     def tag
-      image.split(':')[1] unless image.nil?
+      image.split(':')[-1] unless image.nil? || !image.index(':')
```

Comment:
Splitting on the colon like this could potentially introduce a problem if a non-default repohost was present but no tag was present. For example, if we received `docker ps` output that included:  `repo.example.com:5000/ubuntu`, the tag we'd return would be `5000/ubuntu`

I'd rather see us treat the non-default repo host as a standalone unit and split on `/`, and we need to fix both `#repo` and `#tag`... perhaps something like this:

```ruby
def repo
  return if image_name_from_image.nil?
  image_name_from_image.split(':')[0]
end

def tag
  return if image_name_from_image.nil?
  image_name_from_image.split(':')[1]
end

def image_name_from_image
  return if image.nil?
  # possible image names include:
  #   alpine
  #   ubuntu:14.04
  #   repo.example.com:5000/ubuntu
  #   repo.example.com:5000/ubuntu:1404
  image.include?('/') ? image.split('/')[1] : image
end

```

What do you think, @mattlqx?